src/components/footer/www/footer.jsx

```diff
@@ -20,8 +20,8 @@ const interwikiMapping = {
     hu: 'https://hu.scratch-wiki.info/',
     fr: 'https://fr.scratch-wiki.info/'
 };
-var wikiLink = null;
-if (this.props.intl.locale in interwikiMapping) {
+const wikiLink = null;
+if (props.intl.locale in interwikiMapping) {
```

Comment:
props is only available inside `const Footer =`

Reply:
Oh, thanks.